packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperPlotNav.tsx

```diff
@@ -273,6 +273,30 @@ function QueryWrapperPlotNavContents(props: QueryWrapperPlotNavContentsProps) {
   )
 }
 
+export const getQueryRequest = ({
```

Comment:
Refactor for reuse

Reply:
Where is it reused?

Reply:
It was reused during development, and now it's not!  I'll revert